TypeCobol.Test/Compiler/Parser/ResultFiles/Programs/EmptyPGM.txt

```diff
@@ -1,3 +1 @@
-﻿--- Diagnostics ---
-Line 0[1,0] <27, Error, Syntax> - Syntax error : no viable alternative at input '<EOF>' RuleStack=cobolCompilationUnit,  OffendingSymbol=[1,0:]<EndOfFile>
-
+﻿
```

Comment:
Because of the new grammar `(cobolProgram | cobolClass)* EOF;` ANTLR does not generate an error on empty file. Also because it's an empty file, we can't add a diagnostic. Diagnostic are always associated to a line or at least a token, however, we don't have any line or token... 

I suggest to throw an exeption in FileCompiler, if TextDocument.Lines is null. It means that the empty test needs to be moved as a CLI test in order to get the exception as a diagnostic in error file. 

Do you agree ?